<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-LLRT0RU" failures="2" tests="2" name="Test(failed)" time="77.366" errors="0" timestamp="2025-06-24T11:33:07 IST">
  <testcase classname="testcases.TC002_Logintest" name="verify_login" time="56.345">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testcases.TC002_Logintest.verify_login(TC002_Logintest.java:29)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- verify_login -->
  <testcase classname="testcases.TC001_Accountregistrationpage" name="verify_account_registration" time="15.515">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testcases.TC001_Accountregistrationpage.verify_account_registration(TC001_Accountregistrationpage.java:52)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- verify_account_registration -->
</testsuite> <!-- Test(failed) -->
